src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -76,21 +76,25 @@
            on-import (:on-import props)]
        (assert selected-config (str "Missing a selected configuration: " props))
        (assert on-import (str "Missing on-import handler: " props))
-       (assert workspace-id (str "Missing workspace-id: " props))
        [:div {}
-        (create-formatted-header "Import Method Configuration")
+        (create-formatted-header
+          (if (:workspace-id props)
+          "Import Method Configuration"
+          "Browse Method Configuration"))
         (for [[k v] {"Name" selected-conf-name
                      "Namespace" selected-conf-namespace
                      "Snapshot Id" selected-conf-snapshot-id}]
           (create-formatted-label-text k v))
+        (when (:workspace-id props)
         (for [[k v] {"Destination Name"
                      (style/create-text-field {:defaultValue selected-conf-name :ref "destinationName"})
                      "Destination Namespace"
                      (style/create-text-field {:defaultValue selected-conf-namespace :ref "destinationNamespace"})}]
           (create-formatted-label-textfield k v))
-        (render-import-button props refs)
-        [:span {:style {:marginLeft "0.5em"}}
-         [comps/Button {:text "Back" :onClick #((:on-back props))}]]]))})
+          (render-import-button props refs))
+        [:span {:style {:marginLeft (if (:workspace-id props)"0.5em" "0.0em")}}
```

Comment:
re " I really don't like the placement of this back button. "
What is the "X" button in the upper right is modified to have *two* functions.  The first function, done when viewing the table, is leaving the window altogether.  The second function, when viewing a selected MC (either for browse or for import)  is to have the same functionality as a back button.  The "flip" side of this coin is that the window would necessitate *two* clicks at most to leave it (either import or a two-step back-out).  What do you think?